src/main/java/com/minecolonies/coremod/colony/Colony.java

```diff
@@ -1947,6 +1992,16 @@ public void decreaseOverallHappiness(double amount)
         return new HashMap<>(wayPoints);
     }
 
+    public List<EntityLivingBase> getGuardTargets()
+    {
+        return new ArrayList<>(guardTargets);
+    }
+
+    public void setGuardTargets(final List<EntityLivingBase> targets)
+    {
+        guardTargets = targets;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Store a copy of "targets". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS2384)